run.ps1

```diff
@@ -5,6 +5,9 @@ $restorePackagesPath = Join-Path $PSScriptRoot "packages"
 # We do not want to run the first-time experience.
 $env:DOTNET_SKIP_FIRST_TIME_EXPERIENCE = 1
 
+# Don't resolve runtime, shared framework, or SDK from other locations to ensure build determinism
+$env:DOTNET_MULTILEVEL_LOOKUP=0
```

Comment:
can you also add this in run.sh to keep things consistent.

Reply:
Yes. Thanks